azurerm/resource_arm_key_vault_key.go

```diff
@@ -90,6 +92,22 @@ func resourceArmKeyVaultKey() *schema.Resource {
 				},
 			},
 
+			"curve": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(keyvault.P256),
+					string(keyvault.P384),
+					string(keyvault.P521),
+					string(keyvault.SECP256K1),
+				}, false),
+				// TODO: the curve name should probably be mandatory for EC in the future,
+				// but handle the diff so that we don't break existing configurations and
+				// imported EC keys
+				DiffSuppressFunc: ignoreEmptyCurveChanges,
```

Comment:
It's been a long time, but I think it was something like that. I believe it was also out of caution in case someone had imported an EC key into terraform, though admittedly that'd be a rare case. Since the tests pass, it sounds like that's been fixed on the API end.